extensions/sql-migration/src/api/utils.ts

```diff
@@ -781,6 +849,11 @@ export async function getBlobLastBackupFileNamesValues(blobs: azureResource.Blob
 		|| [{ name: '', displayName: constants.NO_BLOBFILES_FOUND }];
 }
 
+export async function getBlobFolderValues(folders: string[]): Promise<CategoryValue[]> {
+	return folders?.map(folder => { return { name: folder, displayName: folder }; })
```

Comment:
this method doesn't seem to have async calls in it?

Reply:
good catch, to be fair none of these seem to have async calls either actually, will update them all to remove async